frontend/packages/client/src/pages/index.js

```diff
@@ -32,6 +32,9 @@ export default function AppPages() {
             <Route exact path="/community/create">
               <CommunityCreate />
             </Route>
+            <Route path="/community/:communityId/proposal/:proposalId">
+              <Proposal />
+            </Route>
             <Route path="/community/:communityId/edit" exact={true}>
```

Comment:
we have three routes starting with the same path, I'm holding to create a separate folder with the nested route config structure, maybe if this grows we should do that